src/Chapter2.hs

```diff
@@ -619,9 +624,13 @@ Write a function that takes elements of a list only in even positions.
   least 2. Alternatively, you can use the "Recursive go" pattern.
 
 >>> takeEven [2, 1, 3, 5, 4]
-[2,3,4]
+[2,4]
 -}
-takeEven = error "takeEven: Not implemented!"
+takeEven :: [Int] -> [Int]
+takeEven [] = []
+takeEven (x:xs) = isEven x ++ takeEven xs
+    where isEven :: Int -> [Int]
+          isEven n = if even n then [n] else []
```

Comment:
Here, what I meant is to take the elements on the even positions 🙂 
I see that what you are doing is to take even numbers, which is fine!

In this case, you could use the `filter` function and `even` function together.
Also in the current solution, there is no need to introduce `isEven`, you can use `even` and `:` instead of concatinating the lists 🙂